src/components/ha-help-tooltip.ts

```diff
@@ -0,0 +1,42 @@
+import { mdiHelpCircle } from "@mdi/js";
+import "@polymer/paper-tooltip/paper-tooltip";
+import {
+  css,
+  customElement,
+  html,
+  LitElement,
+  property,
+  TemplateResult,
+} from "lit-element";
+import "./ha-svg-icon";
+
+@customElement("ha-help-tooltip")
+export class HaHelpTooltip extends LitElement {
+  @property() public label!: string;
+
+  @property() public position = "top";
+
+  protected render(): TemplateResult {
+    return html`
+      <ha-svg-icon id="HelpIcon" .path=${mdiHelpCircle}></ha-svg-icon>
+      <paper-tooltip for="HelpIcon" offset="4" .position=${this.position}
```

Comment:
This should not be needed if they share the same parent.
```suggestion
      <ha-svg-icon .path=${mdiHelpCircle}></ha-svg-icon>
      <paper-tooltip offset="4" .position=${this.position}
```